milli/src/asc_desc.rs

```diff
@@ -7,26 +7,29 @@ use serde::{Deserialize, Serialize};
 use thiserror::Error;
 
 use crate::error::is_reserved_keyword;
+use crate::search::facet::ParseGeoError;
 use crate::{CriterionError, Error, UserError};
 
 /// This error type is never supposed to be shown to the end user.
 /// You must always cast it to a sort error or a criterion error.
 #[derive(Debug)]
 pub enum AscDescError {
-    InvalidLatitude,
-    InvalidLongitude,
+    GeoError(ParseGeoError),
```

Comment:
Hey, this is not mandatory, if you don't want/don't have the time just tell me and I'll do it later.

But it would be nice to use thiserror on this error structure

Reply:
I'll take care of it and clippy warnings. 
Thank you for reviewing 🙂